src/bun.js/test/expect.zig

```diff
@@ -1377,6 +1377,282 @@ pub const Expect = struct {
         return .zero;
     }
 
+    pub fn toContainEntry(
+        this: *Expect,
+        globalObject: *JSGlobalObject,
+        callFrame: *CallFrame,
+    ) JSValue {
+        defer this.postMatch(globalObject);
+        const thisValue = callFrame.this();
+        const arguments_ = callFrame.arguments(1);
+        const arguments = arguments_.slice();
+
+        if (arguments.len < 1) {
+            globalObject.throwInvalidArguments("toContainEntry() takes 1 argument", .{});
+            return .zero;
+        }
+
+        incrementExpectCallCounter();
+
+        const expected = arguments[0];
+        if (!expected.jsType().isArray()) {
+            globalObject.throwInvalidArgumentType("toContainEntry", "expected", "array");
+            return .zero;
+        }
+        expected.ensureStillAlive();
+        const value: JSValue = this.getValue(globalObject, thisValue, "toContainEntry", "<green>expected<r>") orelse return .zero;
+
+        const not = this.flags.not;
+        var pass = false;
+
+        if (!value.isUndefinedOrNull()) {
+            const key = expected.getIndex(globalObject, 0);
+            const key_string = key.toBunString(globalObject);
+            defer key_string.deref();
+
+            const property_value = expected.getIndex(globalObject, 1);
+            const accessed_property_value = value.getOwn(globalObject, key_string) orelse JSValue.undefined;
```

Comment:
I think the `orelse` case should set `pass = false`. Currently this test will pass when it should fail:
```js
expect({}).toContainEntry([ a, 1 ]);
```

Reply:
Applies similarly to `toContainEntries`, `toContainAllEntries`, and `toContainAnyEntries`